test-cases/artifacts/ami.yaml

```diff
@@ -0,0 +1,15 @@
+ami_id_db_scylla: 'ami-067787e9ea5433c78'
+aws_root_disk_size_db: 50
+cluster_backend: 'aws'
+instance_type_db: 'i3.large'
```

Comment:
The instance_type should be configurable, and it supports to multiple instance types in parallel.

Reply:
Is it required to run same AMI on different instance types just to verify that AMI packaged well?

Reply:
this can be added to the pipeline only

Reply:
cause we have specific code per type in the AMI. different types are tested.

It's true we could cover it in some unit test easily. But we need the option to verify on different types

Reply:
done